server.elm

```diff
@@ -2156,39 +2176,140 @@ editionscript = """
             ));
         }
       }
+
       let highlightsubmit = function() {
         let attrName = this.parentElement.parentElement.querySelector("[name=name]").value;
         this.parentElement.parentElement.querySelector("button").disabled =
           attrName === "" || attrName.trim() !== attrName
       }
+
       if(clickedElem && clickedElem.nodeType === 1) {
-        //      interactionDiv.append(el("div", {}, "Add an attribute:"));
+        // interactionDiv.append(el("div", {}, "Add an attribute:"));
         keyvalues.append(
           el("div", {"class": "keyvalue keyvalueadder"}, [
-             el("span", {}, el("input", {"type": "text", placeholder: "key", value: "", name:"name"}, [], {onkeyup: highlightsubmit})),
-             el("span", {}, el("input", {"type": "text", placeholder: "value", value: "", name:"value"}, [], {onkeyup: highlightsubmit})),
-             el("div", {"class":"modify-menu-icon", title: "Add this name/value attribute"}, [], {innerHTML: plusSVG,
-               disabled: true,
-               onclick() {
-                 clickedElem.setAttribute(
-                   this.parentElement.querySelector("[name=name]").value,
-                   this.parentElement.querySelector("[name=value]").value);
-                 editor_model.clickedElem = clickedElem;
-                 updateInteractionDiv();
-             }})]));
+            el("span", {}, el("input", {"type": "text", placeholder: "key", value: "", name: "name"}, [], {onkeyup: highlightsubmit})),
+            el("span", {}, el("input", {"type": "text", placeholder: "value", value: "", name: "value"}, [], {onkeyup: highlightsubmit})),
+            el("div", {"class":"modify-menu-icon", title: "Add this name/value attribute"}, [], {innerHTML: plusSVG,
+              disabled: true,
+              onclick() {
+                clickedElem.setAttribute(
+                  this.parentElement.querySelector("[name=name]").value,
+                  this.parentElement.querySelector("[name=value]").value
+                );
+                editor_model.clickedElem = clickedElem;
+                updateInteractionDiv();
+              }
+            })
+          ])
+        );
+      }
+
+      function uploadImagesAtCursor(files, srcName) {
+        for (var i = 0, file; file = files[i]; i++) {
+          var targetPathName =  editor.getStorageFolder(file) + file.name;
+          editor.uploadFile(targetPathName, file, (targetPathName, file) => {
+            document.getElementsByClassName("keyvalue")[0].children[1].children[0].setAttribute("value", file.name);
```

Comment:
Please replace `document.getElementsByClassName("keyvalue")[0].children[1].children[0]` by document.getElementById("...") and set a unique id to the input holding the value